File: 230676-Products-Service/GESINV.ProductsService/GESINV.ProductsService.Dominio/Venta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GESINV.ProductsService.Dominio
{
    public class Venta
    {
        public Guid Id { get; set; }
        public string NombreCliente { get; set; }
        public DateTime FechaVenta { get; set; }
        public decimal MontoTotalEnPesos { get; set; }
        public bool Programada { get; set; }
        public bool Realizada { get; set; }
        public Guid EmpresaId { get; set; }
        public List<DetalleVentaProducto> DetallesVentasProductos { get; set; }
    }
}

File: 230676-API-Gateway-Service/GESINV.ApiGateway/modelsout/venta.go
package modelsout

type Venta struct {
	Id                 string                    `json:"id"`
	NombreCliente      string                    `json:"nombreCliente"`
	FechaVenta         string                    `json:"fechaVenta"`
	MontoTotalEnPesos  float64                   `json:"montoTotalEnPesos"`
	Programada         bool                      `json:"programada"`
	Realizada          bool                      `json:"realizada"`
	ProductosYCantidad []ProductoNombreYCantidad `json:"productosYCantidad"`
}
